backend/api/communication/src/main/java/co/airy/core/api/communication/Mapper.java

```diff
@@ -18,10 +20,13 @@
 public class Mapper {
 
     public ConversationResponsePayload fromConversation(Conversation conversation) {
+
         return ConversationResponsePayload.builder()
                 .channel(ChannelPayload.builder()
                         .id(conversation.getChannelId())
-                        .name(conversation.getChannel().getName())
+                        // TODO https://github.com/airyhq/airy/issues/909
```

Comment:
So to get the metadata here we would have to change the topology the same way as required for #909. I will default the name on the frontend and we finish this up after doing #909